core/controller/src/main/scala/whisk/core/controller/Meta.scala

```diff
@@ -448,6 +449,10 @@ trait WhiskMetaApi extends Directives with PostActionActivation {
                 // blocking invoke which got queued instead
                 complete(Accepted, JsObject("code" -> transid.id.toJson))
 
+            case Failure(t: BlockingInvokeTimeout) =>
```

Comment:
This will give us the tid for api-experimental meta calls that go beyond the limit timeout. 👍